pandas/core/arrays/base.py

```diff
@@ -634,6 +634,11 @@ class ExtensionOpsMixin(object):
     """
     A base class for linking the operators to their dunder names
     """
+
+    @classmethod
```

Comment:
ideally should add doc-strings & to the top-level docs, as this is API

Reply:
OK.  I think I'll revert this part of the PR as out-of-scope and do the docs bit separately.